Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonCustomPaletteBase.cs

```diff
@@ -2249,35 +2249,44 @@ public void ImportWithUpgrade(Stream stream)
 
         private void PerformUpgrade(Stream stream)
         {
-            using var reader = new StreamReader(stream);
-            var end = reader.ReadToEnd();
-            reader.Close();
-
-            using (var currentSupportedPaletteSchemaReader = new StreamReader(PaletteSchemaResources.CurrentSupportedPaletteSchema))
+            try
             {
-                using (var xslTextReader = XmlReader.Create(currentSupportedPaletteSchemaReader))
+                using var reader = new StreamReader(stream);
```

Comment:
What do these three lines do? As end is replaced below without being used before?

Reply:
Was from this PR (originally) https://github.com/Krypton-Suite/Standard-Toolkit/pull/1077/files#diff-dbc2716418e699c707d51d10fc7acb7c70ece82d5532003fbadfa034adacc60c

Reply:
Not sure what you mean, it's in the code base I reviewed.

Reply:
> Not sure what you mean, it's in the code base I reviewed.

@Smurf-IV Not sure why, but it looks differently on mine?

![image](https://github.com/user-attachments/assets/b1f12c3d-b849-4428-bc9c-e4290e492c4f)